cmd/entrypoint/main.go

```diff
@@ -30,15 +30,15 @@ import (
 
 	"github.com/containerd/containerd/platforms"
 	"github.com/tektoncd/pipeline/cmd/entrypoint/subcommands"
-	featureFlags "github.com/tektoncd/pipeline/pkg/apis/config"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
-	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
-	"github.com/tektoncd/pipeline/pkg/credentials"
-	"github.com/tektoncd/pipeline/pkg/credentials/dockercreds"
-	"github.com/tektoncd/pipeline/pkg/credentials/gitcreds"
+//	featureFlags "github.com/tektoncd/pipeline/pkg/apis/config"
+	//"github.com/tektoncd/pipeline/pkg/apis/pipeline"
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1/types"
+//	"github.com/tektoncd/pipeline/pkg/credentials"
+//	"github.com/tektoncd/pipeline/pkg/credentials/dockercreds"
+//	"github.com/tektoncd/pipeline/pkg/credentials/gitcreds"
```

Comment:
Why are these commented out? If they need to be removed, please do so instead of commenting them.

Reply:
I think this is temporary (to check the zap symbols) ? @PuneetPunamiya

Reply:
Yeah it's temporary thingy, I'm working on the patch, will keep the pr on hold for now

Reply:
Ah, got it, sorry I didn't realise 🙏